packages/components/tests/dummy/app/templates/components/form/radio-card.hbs

```diff
@@ -0,0 +1,625 @@
+      <p><em>Notice: the attributes will be applied to the
+          <code class="dummy-code">&lt;input type="radio"&gt;</code>
+          element. This means you can use all the standard HTML attributes of the
+          <code class="dummy-code">&lt;input type="radio"&gt;</code>
+          element and all the usual Ember techniques for event handling, validation, etc.</em></p>
+      <p><em>Some examples of HTML attributes that you will likely use:
+          <code class="dummy-code">id</code>,
+          <code class="dummy-code">disabled</code>
+          (<a
+            href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#attributes"
+            rel="noopener noreferrer"
+          >see whole list here</a>) and some examples of Ember modifiers:
+          <code class="dummy-code">&lcub;&lcub;on "click" [do something]&rcub;&rcub;</code>,
+          <code class="dummy-code">&lcub;&lcub;on "change" [do something]&rcub;&rcub;</code>.
+        </em></p>
+    </dd>
+  </dl>
+  <h5 class="dummy-h5">Contextual components</h5>
+  <p class="dummy-paragraph" id="component-api-form-radio-field-contextual-components"><code
+      class="dummy-code"
+    >Icon</code>,
+    <code class="dummy-code">Label</code>,
+    <code class="dummy-code">Badge</code>,
+    <code class="dummy-code">Description</code>
+    and
+    <code class="dummy-code">Generic</code>
+    content are passed to the
+    <code class="dummy-code">RadioCard</code>
+    as contextual components.</p>
+  <dl class="dummy-component-props" aria-labelledby="component-api-form-radio-field-contextual-components">
+    <dt>&lt;[R].Icon&gt;</dt>
+    <dd>
+      <p>It yields an icon inside the card container.</p>
+      <p>For details about its API check the
+        <a href="https://flight-hashicorp.vercel.app/engineering" target="_blank" rel="noopener noreferrer"><code
+            class="dummy-code"
+          >FlightIcon</code></a>
+        component.</p>
+    </dd>
+    <dt>&lt;[R].Label&gt; <code>yielded component</code></dt>
+    <dd>
+      <p>It is a container that yields its content emphasised inside the card.</p>
+    </dd>
+    <dt>&lt;[R].Badge&gt;</dt>
+    <dd>
+      <p>It is a badge inside the card container.</p>
+      <p>For details about its API check the
+        <LinkTo @route="components.badge"><code class="dummy-code">Badge</code></LinkTo>
+        component.</p>
+    </dd>
+    <dt>&lt;[R].Description&gt; <code>yielded component</code></dt>
+    <dd>
+      <p>It is a container that yields its content inside the card.</p>
+      <p>The content can be a simple string or a more complex/structured one, in which case it inherits the text style.</p>
+    </dd>
+    <dt>&lt;[R].Generic&gt; <code>yielded component</code></dt>
+    <dd>
+      <p>It is a container that yields its content inside the card.</p>
+      <p>The content does not inherit any styles and can be customized as desired.</p>
```

Comment:
Should we mention that, no matter what the order in the declaration in code, the `generic` content appears always at the bottom?

Reply:
was the thumbs up something todo, or just an ACK?

Reply:
I tried capturing [the rigid order of arguments (including `Generic` here](https://github.com/hashicorp/design-system/blob/alex-ju/add-radio-card-component/packages/components/tests/dummy/app/templates/components/form/radio-card.hbs#L104-L113)